README.md

```diff
@@ -146,3 +152,7 @@ Sending move commands to the desk seems to make the motors run for about one sec
 The height values the desk provides are in 10ths of a millimetre, and correspond to the height above the desks lowest setting i.e. if you lower the desk as far as it will go then the desk will report its height as being zero. The minimum raw height value is zero and the maximum height is 6500. This corresponds to a range of 620mm to 1270mm off the floor.
 
 The desk appears to be pretty good at not doing anything stupid if you send it stupid commands. It won't try to go below the minimum height or above the maximum height and it doesn't do much if you send lots of commands in quick succession. The usual hit detection works, and it will stop moving if it hits an object and will not respond to further commands until a stop command is sent.
+
+## Troubleshooting
```

Comment:
Please move to the existing Troubleshooting section

Reply:
Oops!